xscns/xs_nue_O16_Suzuki2018.dat

```diff
@@ -0,0 +1,1005 @@
+     -1.632301         7.72583e-04       0          0          0          0          0     
+     -1.631000         7.94262e-04       0          0          0          0          0     
+     -1.629699         8.16555e-04       0          0          0          0          0     
+     -1.628397         8.39470e-04       0          0          0          0          0     
+     -1.627096         8.63015e-04       0          0          0          0          0     
+     -1.625795         8.87203e-04       0          0          0          0          0     
+     -1.624494         9.12042e-04       0          0          0          0          0     
+     -1.623193         9.37546e-04       0          0          0          0          0     
+     -1.621892         9.63726e-04       0          0          0          0          0     
+     -1.620591         9.90594e-04       0          0          0          0          0     
+     -1.619290         1.01816e-03       0          0          0          0          0     
+     -1.617989         1.04645e-03       0          0          0          0          0     
+     -1.616688         1.07546e-03       0          0          0          0          0     
+     -1.615387         1.10521e-03       0          0          0          0          0     
+     -1.614086         1.13571e-03       0          0          0          0          0     
+     -1.612785         1.16698e-03       0          0          0          0          0     
+     -1.611484         1.19904e-03       0          0          0          0          0     
+     -1.610183         1.23189e-03       0          0          0          0          0     
+     -1.608882         1.26555e-03       0          0          0          0          0     
+     -1.607581         1.30003e-03       0          0          0          0          0     
+     -1.606280         1.33536e-03       0          0          0          0          0     
+     -1.604979         1.37154e-03       0          0          0          0          0     
+     -1.603678         1.40858e-03       0          0          0          0          0     
+     -1.602377         1.44651e-03       0          0          0          0          0     
+     -1.601076         1.48534e-03       0          0          0          0          0     
+     -1.599775         1.52509e-03       0          0          0          0          0     
+     -1.598474         1.56576e-03       0          0          0          0          0     
+     -1.597173         1.60738e-03       0          0          0          0          0     
+     -1.595872         1.64996e-03       0          0          0          0          0     
+     -1.594571         1.69352e-03       0          0          0          0          0     
+     -1.593270         6.63682e+01       0          0          0          0          0     
+     -1.591969         1.79051e-03       0          0          0          0          0     
```

Comment:
I’ve noticed one irregularity here at log10(E) = -1.5932769 (corresponding to 25.511 MeV). This matches the excitation energy `E_{x,3}` in the fit; so I suspect it’s a numerical issue. I’ll investigate; let’s wait before we merge this.